db/migrate/20251121092228_create_notify_send_requests.rb

```diff
@@ -0,0 +1,10 @@
+class CreateNotifySendRequests < ActiveRecord::Migration[8.0]
```

Comment:
In general, we do migrations separately. We can do all the reviews here with everything, but can we put the migration, blocklist, and schema into a first pr?